presto-parser/src/main/java/io/prestosql/sql/tree/Format.java

```diff
@@ -25,31 +25,46 @@
 public class Format
         extends Expression
 {
+    private final Expression format;
     private final List<Expression> arguments;
+    private final List<Expression> expressions;
```

Comment:
The change to the Format AST node is unrelated an unnecessary to fix this issue. This is all that's needed: https://github.com/prestosql/presto/pull/3856

Reply:
Yes, I agree the refactor is unnecessary to fix this issue, but I did the refactor during fixing the issue since I thought this refactor could potentially improve the readability

Reply:
Let's separate the two topics and focus on the bug fix first. It think there are a couple of issues with the refactoring that we need to work through, but it will be easier to look at them without mixing concerns.

Reply:
I'm OK to separate them to different topics, will take the refactor out and create another PR